commands/release/release.go

```diff
@@ -255,19 +257,29 @@ func run(cmd *cobra.Command, args []string) error {
 		return stacktrace.Propagate(err, "An error occurred while updating the changelog file at '%s'", changelogFilepath)
 	}
 
-	logrus.Infof("Committing changes locally...")
-	// Commit pre-release changes
-	// These are the only files that should get changed during the pre-release
-	// &git.AddOptions{All: true} ignores git ignore and adds `kurtosis_version`
-	// TODO figure out a way to use All while respecting the .gitignore for `kurtosis_version`
-	pathsToAdd := []string{relChangelogFilepath, relAPIFolderPathForKurtosisMonorepo, typeScriptPackageJSONForMinimalGRPCServer}
-
-	for _, pathToAdd := range pathsToAdd {
-		err = worktree.AddWithOptions(&git.AddOptions{Glob: pathToAdd})
-		if err != nil {
-			logrus.Warnf("An error occurred while adding '%v' does it even exist?", pathToAdd)
-			logrus.Warn("You are probably hitting https://github.com/kurtosis-tech/kudet/issues/22")
+	logrus.Infof("Populating exlcudes for the worktree by parsing the .gitignore file")
+	gitIgnoreFile, err := os.Open(gitIgnoreFileRelPath)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred while reading the '%v' file", gitIgnoreFileRelPath)
+	}
+	defer gitIgnoreFile.Close()
+
+	gitIgnoreFileScanner := bufio.NewScanner(gitIgnoreFile)
+	// split the file by lines
+	gitIgnoreFileScanner.Split(bufio.ScanLines)
+	for gitIgnoreFileScanner.Scan() {
+		pattern := gitIgnoreFileScanner.Text()
+		if isWhiteSpaceOrComment(pattern) {
+			continue
 		}
+		var emptyDomain []string = nil
```

Comment:
Seems like this is a constant, and should be in the `var` section at the top per https://github.com/kurtosis-tech/kurtosis-internal-docs/blob/master/go-style-guide.md#use-the-var-section-for-constant-maps-lists-and-structs